pycqed/analysis/fitting_models.py

```diff
@@ -51,12 +51,11 @@ def Lorentzian(f, A, offset, f0, kappa):
     return val
 
 
-def TwinLorentzFunc(f, amplitude_a, amplitude_b, center_a, center_b,
-                    sigma_a, sigma_b, background=0):
+def TwinLorentzFunc(f, A_gf_over_2, A, f0_gf_over_2, f0,
+                    kappa_gf_over_2, kappa, background=0):
```

Comment:
Could you add some docstring explaining the naming of these values? I guess the naming makes sense but it is hard to understand if you read this without context.